storage/storage.go

```diff
@@ -181,11 +184,38 @@ func initAndGetDriver(configuration Configuration) (driverType DBDriver, driverN
 
 // Init method is doing initialization like creating tables in underlying database
 func (storage DBStorage) Init() error {
+	// Perform all defined migrations on the database.
 	if err := migration.InitInfoTable(storage.connection); err != nil {
 		return err
 	}
+	if err := migration.SetDBVersion(storage.connection, migration.GetMaxVersion()); err != nil {
+		return err
+	}
+
+	// Read clusterName:LastChecked dictionary from DB.
+	rows, err := storage.connection.Query("SELECT cluster, last_checked_at FROM report")
+	if err != nil {
+		return err
+	}
+
+	for rows.Next() {
+		var (
+			clusterName types.ClusterName
+			lastChecked time.Time
+		)
 
-	return migration.SetDBVersion(storage.connection, migration.GetMaxVersion())
+		if err := rows.Scan(&clusterName, &lastChecked); err != nil {
+			_ = rows.Close()
```

Comment:
maybe would be better to log an error here

Reply:
Ok, is it better like this?